packages/block-editor/src/components/multi-selection-inspector/index.js

```diff
@@ -20,27 +19,20 @@ export default function MultiSelectionInspector() {
 			blocks: getMultiSelectedBlocks(),
```

Comment:
```suggestion
			count: getSelectedBlockCount(),
```

The component only needs a selected block count for rendering.

Reply:
Yeah, it's better that way. Thanks!